world_nodes: {
  reception: {
    x: 2,
    y: 6
  },

  store_1: {
    x: 7,
    y: 12
  },

  store_2: {
    x: -7,
    y: 12
  },

  doctor_off_1: {
    x: 1,
    y: -4
  },

  doctor_off_2: {
    x: -1,
    y: -4
  },

  washroom_1: {
    x: 2,
    y: -11
  },

  washroom_2: {
    x: -2,
    y: -11
  },

  ot_1: {
    x: 9,
    y: 11
  },

  ot_2: {
    x: -9,
    y: 11
  },

  ot_3: {
    x: 9,
    y: 5
  },

  ot_4: {
    x: -9,
    y: 5
  },

  ot_5: {
    x: 3,
    y: -16
  },

  ot_6: {
    x: -3,
    y: -16
  },

  ward_1: {
    x: 7,
    y: -9
  },

  ward_2: {
    x: -7,
    y: -9
  },

  ward_3: {
    x: 7,
    y: -23
  },

  ward_4: {
    x: -7,
    y: -23
  },

  ward_5: {
    x: 5,
    y: -27
  },

  wait: {
    x: -8,
    y: -26
  },

  dock_1: {
    x: -5,
    y: -34
  },

  dock_2: {
    x: 0,
    y: 1.5
  },

  dock_3: {
    x: 0,
    y: -14.5
  },
}